### ISSUE TITLE
ASHRAE Revised Clear Sky "tau" Model

### ISSUE THREAD

**Issue by mjroth** [NREL/EnergyPlus#6864]
Issue overview
--------------
The ASHRAE "tau model" uses the original but obsolete coefficients for the air mass exponents from HOF 2009.  The last two updates (HOF 2013 and HOF 2017) have published improved coefficients.

The pertinent lines in WeatherManager.cc for ASHRAETauModel:
```
AB = 1.219 - 0.043 * TauB - 0.151 * TauD - 0.204 * TauB * TauD;
AD = 0.202 + 0.852 * TauB - 0.007 * TauD - 0.357 * TauB * TauD;
```

Should read:
```			
AB = 1.454 - 0.406 * TauB - 0.268 * TauD + 0.021 * TauB * TauD;
AD = 0.507 + 0.205 * TauB - 0.080 * TauD - 0.190 * TauB * TauD;
```

### Details
Reference:  ASHRAE Handbook - Fundamentals, 2017, Chapter 14, pg. 14.10, eqns (19) & (20)

**Comment by mjwitte** [comment]
True, but the coefficients for the air mass exponents are not the only thing that changed.  All of the location-specific data for TauB and TauD also changed. For example, from handbook Ch14, Table 1 for Atlanta, GA, here are taub, taud, and the results of an example calulation for Atlanta:
2009 HOF July taub=0.556, taud=1.779, Eb=751 W/m2, Ed=220 (at solar noon) 737/215 at 3pm EDT
2013 HOF July taub=0.440, taud=2.202, Eb=846, Ed=144 (at solar noon) 834/141 at 3pm EDT
2017 HOF July taub=0.515, taud=2.066, Eb=784, Ed=166 (at solar noon) 770/162 at 3pm EDT

The EnergyPlus ddy files which are available on [energyplus.net](https://energyplus.net/weather) use the 2009 values for taub and taud, at least that's true for [Atlanta TMY3](https://energyplus.net/weather-location/north_and_central_america_wmo_region_4/USA/GA/USA_GA_Atlanta-Hartsfield-Jackson.Intl.AP.722190_TMY3).
```
 ! Atlanta Hartsfield Intl Ap Annual Cooling Design Conditions Wind Speed=4m/s Wind Dir=300
 ! Hottest Month=JUL
 ! Atlanta Hartsfield Intl Ap_GA_USA Annual Cooling (DB=>MWB) .4%, MaxDB=34.4°C MWB=23.5°C
 SizingPeriod:DesignDay,
  Atlanta Hartsfield Intl Ap Ann Clg .4% Condns DB=>MWB,     !- Name
          7,      !- Month
         21,      !- Day of Month
  SummerDesignDay,!- Day Type
       34.4,      !- Maximum Dry-Bulb Temperature {C}
        9.5,      !- Daily Dry-Bulb Temperature Range {C}
<snip>
       ASHRAETau, !- Solar Model Indicator
           ,      !- Beam Solar Day Schedule Name
           ,      !- Diffuse Solar Day Schedule Name
      0.556,      !- ASHRAE Clear Sky Optical Depth for Beam Irradiance (taub)
      1.779;      !- ASHRAE Clear Sky Optical Depth for Diffuse Irradiance (taud)
```
The other ddy files that are part of the EnergyPlus install all reference the 2009 HOF in the comments. I haven't confirmed the numbers, but expect they will match the 2009 data.

So, this is quite messy and will require options to select which vintage of ASHRAETau is to be used. It would be interesting (disturbing?) to see what happens with the wrong mix of equation coefficients and tau values.

**Comment by mjwitte** [comment]
Closed via #6891
